latch_cli/services/k8s/utils.py

```diff
@@ -62,6 +61,9 @@ def with_fragments(doc: graphql.DocumentNode, *fragments: graphql.DocumentNode):
         workflow {
             displayName
         }
+        nfExecutionInfoByExecutionId {
```

Comment:
this won't work for relaunches, will have to get the nfexecutioninfo of the parent execution recursively. 

Or perhaps if we fix the relaunch functionality, none of this would be required.